src/txmempool.cpp

```diff
@@ -163,6 +163,17 @@ util::Result<CTxMemPool::setEntries> CTxMemPool::CalculateAncestorsAndCheckLimit
     int64_t totalSizeWithAncestors = entry_size;
     setEntries ancestors;
 
+    // Package itself is busting mempool limits; should be rejected even if no staged_ancestors exist
```

Comment:
Needs test coverage, maybe in test/functional/mempool_sigoplimit.py? Or a unit test for `CheckPackageLimits`?

Reply:
Added in mempool_sigoplimit.py since this change threads through a few layers